kvgraph/keys.go

```diff
@@ -0,0 +1,120 @@
+package kvgraph
+
+import (
+	"bytes"
+)
+
+var GRAPH_PREFIX []byte = []byte("g")
+var VERTEX_PREFIX []byte = []byte("v")
+var EDGE_PREFIX []byte = []byte("e")
+var SEDGE_PREFIX []byte = []byte("s")
+var DEDGE_PREFIX []byte = []byte("d")
+
+var EDGE_SINGLE byte = 0x01
+var EDGE_BUNDLE byte = 0x02
+
+func GraphPrefix() []byte {
+	return GRAPH_PREFIX
+}
+
+func GraphKey(graph string) []byte {
+	return bytes.Join([][]byte{GRAPH_PREFIX, []byte(graph)}, []byte{0})
+}
+
+func GraphKeyParse(key []byte) string {
+	tmp := bytes.Split(key, []byte{0})
+	graph := string(tmp[1])
+	return graph
+}
+
+func EdgeKey(graph, id, src, dst, label string, etype byte) []byte {
```

Comment:
Oh, I see you already have key builder helpers here. In this case, I recommend adding a `type Key []byte` to make the API more obvious, then returning `Key` from these helpers.

Reply:
All of the underlying driver libraries use []byte, so I'm a little worried about ending up with type conversion code everywhere.

Reply:
Better to have it in internal database code than to make it part of the core public API. Also, if I'm doing this correctly, this shows you won't need to worry about it https://play.golang.org/p/7WQwKWytz2g

Reply:
We can probably do that later. I'd prefer to get this PR in, and deal with that later.